custom-connectors/Datelist/README.md

```diff
@@ -0,0 +1,11 @@
+# Datelist - Microsoft Flow Connector
+Microsoft flow connector for https://datelist.io 
```

Comment:
@aclarembeau `Flow` is an outdated word in the product, Can you please update is with `Power Automate`